administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -340,9 +340,9 @@ const ApplicationCard = ({
   const otherOptionsItemHeight = 36.5
   const menuItems: MenuItemType[] = [
     {
-      name: t('exportPdf'),
+      name: t('printPdf'),
       onClick: () => onPrintApplicationById(application.id),
-      icon: <PrintOutlined sx={{ height: 24, marginRight: 1 }} />,
+      icon: <PrintOutlined sx={{ height: 20, marginRight: 1 }} />,
```

Comment:
we have 20 for other icons
<img width="300" alt="image" src="https://github.com/user-attachments/assets/2ef40f16-59fa-475e-b264-a79ffa38dc3a" />